src/com/android/volley/Request.java

```diff
@@ -699,4 +704,17 @@ public void setJoined(boolean joined) {
     public void expireCache() {
         // no - op
     }
+
+    public boolean isExecuting() {
```

Comment:
Let's get rid of the `isRunning` method in `AirbnbRequest` and use this new one instead.